src/main/java/com/box/sdk/BoxUser.java

```diff
@@ -680,12 +682,10 @@ public BoxFolder.Info transferContent(String destinationUserID) {
      *
      * @return InputStream representing the user avater.
      */
-    public InputStream getAvatar() {
+    public void downloadAvatar(OutputStream outputStream) {
```

Comment:
Since response holds opened connection if user forgets to close it then we are leaking http connections. It is better to use same approach we have with files download where user provides an output stream, we write there and then we can close connection and prevent leaking.